climada/hazard/test/test_base_xarray.py

```diff
@@ -369,9 +367,6 @@ def test_missing_dims(self):
             ds = ds.expand_dims(time=[np.datetime64("2022-01-01")])
             hazard = Hazard.from_xarray_raster(ds, "", "")
             self._assert_default_types(hazard)
-            np.testing.assert_array_equal(
-                hazard.event_name, [np.datetime64("2022-01-01")]
-            )
```

Comment:
Same as above, why not check the new expected value?

Reply:
see above